index.html

```diff
@@ -1109,24 +1124,21 @@ <h2><dfn>Predicted events</dfn></h2>
 
         <section>
             <h2><dfn>Populating and maintaining the coalesced and predicted events lists</dfn></h2>
-            <p>When a <a>PointerEvent</a> is created, run the following steps for each event in the
+            <p>When a <a>PointerEvent</a> is created, user agents SHOULD run the following steps for each event in the
```

Comment:
I wonder if this should only be for trusted pointer events, or whether we should be doing this processing on user constructed PointerEvents' coalesced and predicted points lists.

Reply:
my reading here at the moment is that it's currently requiring this both for trusted and untrusted events, right? are you proposing adding an explicit note about author-generated events that provide their own coalesced/predicted events lists, specifying that user agents SHOULD also go through the lists to ensure this is the case and sanitise whatever the author provided?

we had some previous conundrum with this when it came to the tiltX/tiltY/azimuthAngle/altitudeAngle, and there we decided in the end to require this too for untrusted events - see the note in 4.1

> When an untrusted (synthetic) Pointer Event is created programmatically using the constructor, and only one set of attributes is provided, the complementary set of attributes SHOULD be calculated and initialized by the user agent. If both sets of attributes are provided, no calculation should be performed. If only the value of one of the two attributes is provided, the other attribute SHOULD be initialized to the default value.

Reply:
further, should user agents also sanitize the timestamps that an author has provided, to make sure they match the requirement about monotonically increasing / greater or smaller than the parent event's timestamp?

Reply:
I would have thought we wouldn't attempt to sanitize untrusted events, given that it leads to things like needing to standardize things like the accessor and setter order since all of that is observable when you have constructed your own event.

Reply:
Agreed in today's PEWG meeting (https://www.w3.org/2021/07/21-pointerevents-minutes.html) to add `trusted` here to make sure sanitisation only happens for trusted events. synthetic/untrusted events, we just take whatever the developer provided at face value and don't expect UA to attempt to rewrite/sanitise those lists.